paasta_tools/prune_completed_pods.py

```diff
@@ -40,6 +40,14 @@ def parse_args():
         required=False,
         type=int,
     )
+    parser.add_argument(
+        "-p",
+        "--pending-minutes",
+        help="Minutes since the pod was scheduled. Terminates pods whose phase is Pending based on time since scheduled. "
```

Comment:
nit: can we explicitly concatenate these two strings?

Reply:
Done.